Makefile

```diff
@@ -1,183 +1,397 @@
 DEFAULT_GOAL: help
 SHELL := /bin/bash
 GCLOUD_VERSION := 222.0.0-1
-PWD := $(shell pwd)
+SDROOT := $(shell pwd)
 TAG ?= $(shell git rev-parse HEAD)
 STABLE_VER := $(shell cat molecule/shared/stable.ver)
+VENV := $(shell readlink -m .venv)
```

Comment:
[running through test plan] this is not portable on macOS (dev env needs to work on both mac and linux): `readlink`'s  `-m` option does not exist as mac is using a very old version of `readlink`

Reply:
Removed.